docs/kyma/docs/028-details-deploy-private-registry.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: How to deploy a Docker image from a private registry
+title: Deploying from a private Docker registry
```

Comment:
Go with "Deploy with (...)" to be consistent with our guidelines.

> While gerunds are acceptable in body-level content, DO NOT use gerunds in headings. Use Create a storefront instead of Creating a storefront.